gammapy/datasets/make.py

```diff
@@ -64,3 +71,121 @@ def sigma_energy_theta(energy, theta, sigma):
+    #       then convert to alt az, considering the observatory location
+
+    # TODO: is there a way to comment on the column names?!!!!!
+    # TODO: store obs name as a column or as a header value?!!!
+    # TODO: format times!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    # TODO: alt az at mean time!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    # TODO: restrict to night time? (or dark time?)!!!!!!!!!!!!
+    # TODO: in utils.time
+    #       - a function to convert a certain time to time ref (int + fraction) + seconds after ref
+    #       - a function to convert a certain time to seconds after a given time ref (int + fraction)
+
+    # start time
+    # random points between the start of 2010 and the end of 2014
+    # using the start of 2010 as a reference time for the header of the table
+    # TODO: restrict to night time? (or dark time?)!!!
+    #       if so: take into account that enough time has to be granted for the last observation of the night to finish (~ 30 min)
+    datestart = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    dateend = Time('2015-01-01 00:00:00', format='iso', scale='utc')
+    time_start = Time((dateend.mjd - datestart.mjd)*np.random.random(len(col_obs_id)) + datestart.mjd, format='mjd', scale='utc').iso
+    # TODO: using format "iso" for ~astropy.Time for now; eventually change it
+    # to "fits" after the next astropy stable release (>1.0) is out.
+    col_time_start = Column(name='TSTART', data=time_start)
+    obs_table.add_column(col_time_start)
+
+    # stop time
+    # calculated as TSTART + ONTIME
+    time_stop = Time(obs_table['TSTART']) + obs_table['ONTIME']
+    col_time_stop = Column(name='TSTOP', data=time_stop)
+    obs_table.add_column(col_time_stop)
+
+    # az, alt
+    # random points in a sphere above 45 deg altitude
+    az, alt = sample_sphere(len(col_obs_id), (0, 360), (45, 90), 'degree')
+    az = Angle(az, 'degree')
+    alt = Angle(alt, 'degree')
+    col_az = Column(name='AZ', data=az)
+    obs_table.add_column(col_az)
+    col_alt = Column(name='ALT', data=alt)
+    obs_table.add_column(col_alt)
+
+    # RA, dec
+    # derive from az, alt taking into account that alt, az represent the values at the middle of the observation, i.e. at (TSTART + TSTOP)/2 (or TSTART + (ONTIME/2))
+    az = Angle(obs_table['AZ'])
+    alt = Angle(obs_table['ALT'])
+    obstime = obs_table['TSTART']
+    ##obstime = obs_table['TSTART'] + obs_table['TSTOP']
+    ##obstime = Time(obs_table['TSTART']) + Time(obs_table['TSTOP'])
+    location = observatory_locations[observatory]
+    alt_az_coord = AltAz(az = az, alt = alt, obstime = obstime, location = location)
+    # TODO: make it depend on other pars: temperature, pressure, humidity,...
+    sky_coord = alt_az_coord.transform_to(FK5)
+    ra = sky_coord.ra
+    col_ra = Column(name='RA', data=ra)
+    obs_table.add_column(col_ra)
```

Comment:
I think some of these lines can be combined to get a shorter yet not less readable function:
```python
obs_table.add_column(Column(name='RA', data=skycoord.ra))
```